jenkins/custom-ci/centos-7.9.2009/custom-ci.groovy

```diff
@@ -51,6 +51,12 @@ pipeline {
             description: 'Third Party RPM Version to use.'
         )
 
+        choice(
+            name: 'MOTR_BUILD_MODE',
```

Comment:
Please make `user-mode` as default. As most of the developers will be using K8s deployment.

Reply:
done